src/index.ts

```diff
@@ -41,6 +42,8 @@ export const signerMethods = [
 type SignerMethodsTuple = typeof signerMethods;
 type SignerMethods = SignerMethodsTuple[number];
 
+export type ChainGroup = -1 | 0 | 1 | 2 | 3
```

Comment:
I tend to avoid the assumption of 4 groups in our tools whenever it's possible, as we will increase the group number and build L2 chains.

Type side, `ChainGroup = number | undefined` might be less confusing than `-1`. `undefined` for devs and `-1` for the underlying protocol.

There is also an inconsistency as far as I see. Sometimes the code uses `[alephium:4/0, alephium:4/1, alephium:4/2, alephium:4/3]`, sometime the code uses `alephium:4/-1`

Reply:
Sounds great, `undefined` feels a lot more dev friendlier than `-1` :+1: